caluma/caluma_core/events.py

```diff
@@ -1,18 +1,26 @@
 import logging
 from warnings import warn
 
-from django.dispatch import receiver
-
 logger = logging.getLogger(__name__)
 
 
-def on(event, *args, **kwargs):
+def on(event, raise_exceptions=False, *args, **kwargs):
```

Comment:
It will just raise the first exception that occurs, so maybe make this arg singular.

Reply:
True! Fixed it